### PR TITLE
Added book: Functional-Light Javascript

### PR DESCRIPTION
**Pull request by HARDY8118**
## What does this PR do?
Add resource(s)

### Description
Added a free book **Functiona Light JS**

### Why is this valuable (or not)?
This free book by Kyle Simpson helps anyone who wants to understand JavaScript functional programming and functional programming in general.

### How do we know it's really free?
The book is hosted on Kyle Simpson's github account. The paid version is available for someone (link in book's repo) who wants to buy a copy however a free version is also available using the repo itself.

### For book lists, is it a book? For course lists, is it a course? etc.
It is a book.

## Checklist:
- [x ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by eshellman** [comment]
pick on list or another. we don't allow duplicates

**Comment by HARDY8118** [comment]
Can you provide a little more information?

File: books/free-programming-books-subjects.md [hunk 1]
```diff
@@ -700,6 +700,7 @@ Kerridge (PDF) (email address *requested*, not required)
 ### Programming Paradigms
 
 * [Flow based Programming](http://jpaulmorrison.com/fbp/) - J Paul Morrison
+* [Functional-Light JavaScript](https://github.com/getify/Functional-Light-JS) - Kyle Simpson (HTML)
```
Comment (eshellman) [inline books/free-programming-books-subjects.md:703]: ```suggestion```

**Comment by eshellman** [comment]
Sorry to have missed the question, we've had over a thousand PRs.
Thanks, and welcome to free-programming-books!

**Pull request merged**